homeassistant/components/deconz/climate.py

```diff
@@ -187,7 +218,9 @@ async def async_set_hvac_mode(self, hvac_mode: str) -> None:
             raise ValueError(f"Unsupported HVAC mode {hvac_mode}")
 
         data = {"mode": self._hvac_mode_to_deconz[hvac_mode]}
-        if len(self._hvac_mode_to_deconz) == 2:  # Only allow turn on and off thermostat
+        if (
+            len(self._hvac_mode_to_deconz) == 2 and self._device.modelid != "SPZB0001"
```

Comment:
```suggestion
            len(self._hvac_mode_to_deconz) == 2 and self._device.modelid != EUROTRONIC_SPZB0001
```

Reply:
Same here SPZB0001 has only two modes but is not controlled by on off

Reply:
Yes in this case it seems to be necessary